### PR TITLE
PIN-3457 Updated Party Registry Proxy

### PR DESCRIPTION
**Pull request by nttdata-rtorsoli**

### CODE REVIEW

**Review by TonioGela — APPROVED** [review]

File: src/main/scala/it/pagopa/interop/partyregistryproxy/service/impl/PagopaOpenDataServiceImpl.scala [hunk 1]
```diff
@@ -24,7 +24,8 @@ object PagopaOpenDataServiceImpl extends OpenDataService {
           digitalAddress = "apogap@pagopa.it",
           address = "Piazza Colonna 370, Roma",
           zipCode = "00187",
-          origin = INTEROP_ORIGIN
+          origin = INTEROP_ORIGIN,
+          kind = "Pubbliche Amministrazioni"
```
Comment (galales) [inline src/main/scala/it/pagopa/interop/partyregistryproxy/service/impl/PagopaOpenDataServiceImpl.scala]: @beetlecrunch should PagoPa have this kind?

File: src/main/scala/it/pagopa/interop/partyregistryproxy/service/impl/util/DocumentConverter.scala [hunk 2]
```diff
@@ -26,7 +26,8 @@ object DocumentConverter {
       digitalAddress = document.get(DIGITAL_ADDRESS.value),
       address = document.get(ADDRESS.value),
       zipCode = document.get(ZIP_CODE.value),
-      origin = document.get(ORIGIN.value)
+      origin = document.get(ORIGIN.value),
+      kind = "Pubbliche Amministrazioni"
```
Comment (galales) [inline src/main/scala/it/pagopa/interop/partyregistryproxy/service/impl/util/DocumentConverter.scala]: am I missing something, or should this be something like
```suggestion
      kind = document.get(KIND.value)
```

File: src/main/scala/it/pagopa/interop/partyregistryproxy/service/impl/PagopaOpenDataServiceImpl.scala (hunk 1 above)
Comment (beetlecrunch) [reply src/main/scala/it/pagopa/interop/partyregistryproxy/service/impl/PagopaOpenDataServiceImpl.scala]: I asked @marcocipri , but I think that here we should put a custom value, for example (PagoPA)

Comment (beetlecrunch) [reply src/main/scala/it/pagopa/interop/partyregistryproxy/service/impl/PagopaOpenDataServiceImpl.scala]: Let's put "Privato" for Pagopa

**Review by beetlecrunch — APPROVED** [review]

**Review by galales — APPROVED** [review]

**Pull request merged**